cloudformation/service/us-east-1/lambda.yml

```diff
@@ -0,0 +1,67 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: CloudFormation template for Lambda resources
+
+Parameters:
+  Stage:
+    Type: String
+    AllowedValues:
+      - test
+      - preprod
+      - prod
+
+Resources:
+  SecurityHeadersEdgeLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Path: /
+      ManagedPolicyArns:
+        - "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: "AllowLambdaServiceToAssumeRole"
+            Effect: "Allow"
+            Action:
+              - "sts:AssumeRole"
+            Principal:
+              Service:
+                - "lambda.amazonaws.com"
+                - "edgelambda.amazonaws.com"
+
+  SecurityHeadersEdgeLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub security-headers-edge-lambda-${Stage}
+      Role: !GetAtt SecurityHeadersEdgeLambdaRole.Arn
+      Runtime: nodejs12.x
+      Handler: index.handler
+      Timeout: 5
+      Code:
+        ZipFile: |
```

Comment:
Any reason for this to not be a serverless lambda with source code?

Reply:
Mainly just because we haven't been using serverless in the AWS repo as of yet and that would add an extra step with the cloudformation deploy and a serverless deploy. I think the dream would still be to move to CDK to handle everything though.

Reply:
Ah ok cool, guess this is simple enough for now 👍